sessions/forecasting-concepts.qmd

```diff
@@ -31,6 +31,149 @@ The code in this session can be run as an interactive notebook using RStudio, or
 It needs to be run inside the course repository so that the `here()` commands below find the stan model files.
 :::
 
+# Introduction to forecasting as an epidemiological problem, and its relationship with nowcasting and R estimation
+
+See the lecture slides.
+
+# What is forecasting?
+
+Forecasting is the process of making predictions about the future based on past and present data.
+In the context of infectious disease epidemiology, forecasting is the process of predicting the future course of an epidemic based on past and present data.
+Here we focus on forecasting observed data (i.e. onsets) but forecasts can also be made for other quantities of interest such as the number of infections, the reproduction number, or the number of deaths.
+Epidemiological forecasting is closely related to nowcasting and, when using mechanistic approaches, estimation of the reproduction number. In fact, the model we will use for forecasting is the same as the model we used for nowcasting and estimation of the reproduction number. The only difference is that we will extend the model into the future and for the purpose of this session we will remove the nowcasting part of the model.
+
+# Extending a model into the future
+
+The model we introduced in the [renewal session]() the reproduction number using a random walk, then used a discrete renewal process to model the number of infections, and convolved these with a delay distribution to model the number of onsets with Poisson observation error.
+Based on what we found in the [nowcasting session]() this seems like a reasonable model for the data and so we might want to use it to forecast into the future.
+We can do this by extending the model into the future by simulating the model forward in time. 
+This is known as a generative model.
+We actually already introduced a model that could do this in the renewal session (remember `h` from the data list in the renewal session?).
+So what does this look like in code? Lets load in the model again and take a look.
```

Comment:
IMO, I think we should introduce the forecasting piece here and not in the renewal equation session.